test/app.js

```diff
@@ -150,4 +123,13 @@ describe('Smoke test', () => {
     await app.auth.loginFromURI(h.authUris.unregisteredUri);
     should(app.getAccountInfo()).be.rejected();
   });
+
+  it('returns safe_client_libs log path', async () => {
+    const app = createAuthenticatedTestApp();
+    app.logPath().should.be.fulfilled();
```

Comment:
In cases like this one, you need to return the `Promise` you get from `app.logPath()` otherwise the test won't fail if the `Promise` is rejected, you would only get an `unhandled promise` error but the test would still be passing. Please make sure you return `Promises` in all tests.